Forms/HTMLEditor/FileShortcodeProvider.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+namespace SilverStripe\Forms\HtmlEditor;
```

Comment:
Update on previous comment:
Wouldn't it be more logic to put all the Shortcode Parsers under the same namespace? As the interface is defined under `SilverStripe\View\Parsers`, I would suggest moving all providers to the same namespace?